Dayol-iOS/UI/Common/DYModalViewController/DYModalViewController.swift

```diff
@@ -135,7 +135,11 @@ class DYModalViewController: UIViewController {
 
     override func dismiss(animated flag: Bool, completion: (() -> Void)? = nil) {
         dismissContentView(animated: flag) {
-            super.dismiss(animated: false, completion: self.dismissCompeletion)
+            if let completionHandler = self.dismissCompeletion {
```

Comment:
dismissCompletion이랑 dismiss의 completion이랑 왜 분리를 한건지 궁금합니다~

Reply:
self.dismissCompeletion는 제가 정의한 변수인데요.
다이어리 편집 중 툴(모달로 present)를 사용할 때 편의를 위해 사용했는데요,  

modal을 호출하는 쪽에서 dismiss를 호출하지 않고 modal 내부 조작으로 dismiss가 되는 경우를 위해 
dismissCompletion을 따로 정의했었습니다.  

ex) 
![스크린샷 2021-05-24 오전 12 47 21](https://user-images.githubusercontent.com/46941349/119267446-9be3a300-bc29-11eb-94dc-b3d426bdd0b7.png)  

컬러피커에서 색상을 골라보고 변경 -> 모달 뷰의 우측 상단 아래 화살표 모양 버튼 or 상단 빈 공간 터치하면 modalVC에서 dismiss (modalVC 내부 구현) -> 모달 뷰 조작으로 dismiss될 때 편집화면에 선택한 색상 적용 (dismissCompletion)   

우선 현재 pr에선

```suggestion
super.dismiss(animated: false, completion: completion)
self.dismissCompeletion?()
```
일단 이렇게 둘 다 호출해도록 수정해주시면  
추후에 self.dismissCompeletion?()은 제거하는 방향으로 수정해보겠습니다.